word-search-page/word-search.js

```diff
@@ -0,0 +1,36 @@
+//Selectors
+function get(id1) {
+    return document.getElementById(id1);
```

Comment:
**here you select elements by `id`, but actually they don't have an id selector, you gave them a class , Fix this problem.**